Python-Input.md

```diff
@@ -0,0 +1,33 @@
+# Python Input() Function
+
+Many a times, in a program we need some input from the user. Taking inputs from the user makes the program feel a bit interactive. In python, to take input from the user we have a function `input()`. Let's see some examples:
+
+1. When we just want to take the input:
+```python
+# This will just give a prompt without any message
+inp = input()
+```
+[Run Code](https://repl.it/CUqX/0)
```

Comment:
Should I squash the commits?


On Wed, May 25, 2016 at 10:47 AM, Arijit Layek <notifications@github.com>
wrote:

> In Python-Input.md
> <https://github.com/FreeCodeCamp/wiki/pull/1031#discussion_r64516233>:
>
> > @@ -0,0 +1,33 @@
> > +# Python Input() Function
> > +
> > +Many a times, in a program we need some input from the user. Taking inputs from the user makes the program feel a bit interactive. In python, to take input from the user we have a function `input()`. Let's see some examples:
> > +
> > +1. When we just want to take the input:
> > +```python
> > +# This will just give a prompt without any message
> > +inp = input()
> > +```
> > +[Run Code](https://repl.it/CUqX/0)
>
> Don't think you would need a REPL snippet here. It does nothing when you
> run it. Except wait for an input.
>
> @koustuvsinha <https://github.com/koustuvsinha> @atjonathan
> <https://github.com/atjonathan> @Rafase282 <https://github.com/Rafase282>
> what do you guys think?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeCodeCamp/wiki/pull/1031/files/a244bb0f8fade13700ebf3e6900deba88d0e7a61#r64516233>
>